src/array/cpu/concurrent_id_hash_map.h

```diff
@@ -25,18 +25,20 @@ namespace aten {
  *
  * The hash map should be prepared in two phases before using. With the first
  * being creating the hashmap, and then initialize it with an id array which is
- * divided into 2 parts: [`seed ids`, `left ids`]. In result `seed ids` are
- * mapped to [0, num_seed_ids) and `left ids` to [num_seed_ids, num_unique_ids).
- * Notice that mapping order is stable for `seed ids` while not for the left.
+ * divided into 2 parts: [`seed ids`, `left ids`]. `Seed ids` refer to
+ * a set ids chosen as the input for sampling process and `left ids` are the
```

Comment:
left ids -> sampled ids.

sampled ids are the ids new sampled from the process (note the the seed ids might be sampled in the process, but not included in the sampled ids to avoid duplication)

Reply:
Good description, adopt it in the notes.